src/Symfony/Component/Validator/Resources/translations/validators.tr.xlf

```diff
@@ -402,6 +402,31 @@
                 <source>The value of the netmask should be between {{ min }} and {{ max }}.</source>
                 <target>Netmask'in değeri {{ min }} ve {{ max }} arasında olmaldır.</target>
             </trans-unit>
+            <trans-unit id="104">
+               <source>The filename is too long. It should have {{ filename_max_length }} character or less.|The filename is too long. It should have {{ filename_max_length }} characters or less.</source>
+               <target>Dosya adı çok uzun. {{ filename_max_length }} karakter veya daha az olmalıdır.|Dosya adı çok uzun. {{ filename_max_length }} karakter veya daha az olmalıdır.</target>
+            </trans-unit>
+            <trans-unit id="105">
+              <source>The password strength is too low. Please use a stronger password.</source>
+              <target>Şifre gücü çok düşük. Lütfen daha güçlü bir şifre kullanın.</target>
+            </trans-unit>
+            <trans-unit id="106">
+             <source>This value contains characters that are not allowed by the current restriction-level.</source>
+             <target>Bu değer, mevcut kısıtlama seviyesi tarafından izin verilmeyen karakterler içeriyor.</target>
+           </trans-unit>
+           <trans-unit id="107">
+            <source>Using invisible characters is not allowed.</source>
+            <target>Görünmez karakterlerin kullanılması izin verilmez.</target>
+          </trans-unit>
+          <trans-unit id="108">
+           <source>Mixing numbers from different scripts is not allowed.</source>
+           <target>Farklı yazı türlerinden sayıların karıştırılması izin verilmez.</target>
+          </trans-unit>
+          <trans-unit id="109">
+           <source>Using hidden overlay characters is not allowed.</source>
+           <target>Gizli üstü kaplama karakterlerinin kullanılması izin verilmez.</target>
```

Comment:
GPT suggest `Gizli karakterlerin kullanılması izin verilmez.` for this one, the current translation is too literal it says.